src/qml/glacier-contacts.qrc

```diff
@@ -0,0 +1,18 @@
+<RCC>
+    <qresource prefix="/">
+        <file>components/MessagesInterface.qml</file>
+        <file>components/AvatarPickerSheet.qml</file>
+        <file>components/SearchBox.qml</file>
+        <file>components/EditableList.qml</file>
+        <file>components/ContactImportSheet.qml</file>
+        <file>components/ContactCardContentWidget.qml</file>
+        <file>components/DeleteContactDialog.qml</file>
+        <file>api/ContactListDelegate.qml</file>
```

Comment:
API uses another applications like dialer and other

Reply:
Oh I see, that's true.